pkg/web/apps/apps_http_handler_test.go

```diff
@@ -35,9 +37,16 @@ func Test_HttpHandler_GetApps(t *testing.T) {
 	}
 }
 
-func Test_HttpHandler_GetAppsById(t *testing.T) {
+func Test_HttpHandler_GetAppsByID_mock(t *testing.T) {
```

Comment:
What is the difference between the test functions `Test_HttpHandler_GetAppsByID_mock` and `Test_httpHandler_GetAppByID`?

Reply:
old function should not be here.